simplex/test_util.go

```diff
@@ -18,7 +18,11 @@ func newTestValidatorInfo(allVds []validators.GetValidatorOutput) map[ids.NodeID
 	return vds
 }
 
-func newEngineConfig() (*Config, error) {
+func newEngineConfig(numNodes uint64) (*Config, error) {
+	if numNodes == 0 {
+		numNodes = 1 // Ensure at least one node for testing
```

Comment:
I think it's odd to correct input passed to this function. Why not panic instead?